### ISSUE TITLE
PS4 Keyboard Attachment Support

### ISSUE THREAD

**Issue by corydorning** [Jays2Kings/DS4Windows#127]
Is it possible to make this workable with one of the keyboard attachments? Love the ps4 controller because of the touch pad but the xbone keyboard attachment is a huge plus. Would be nice if there was a workable keyboard attachment driver.

I'm using the PS4 ChatBoost by Collective Minds (only backlit one I could find). But obviously doesn't "just work".

**Comment by laundry-96** [comment]
I don't believe there is any current support for the audio jack or extension jack.
